src/components/BookFeatures.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable react/prop-types */
```

Comment:
- [ ] Kindly remove this & solve it manually.

Reply:
Do you have a suggestion that I can use to solve the issue?

Reply:
@Kelvin-Ben , kindly check this one.[LINK](https://blog.logrocket.com/validate-react-props-proptypes/)